src/components/FAQIndividualComponent.jsx

```diff
@@ -7,42 +7,47 @@ function FAQIndividualComponent(props) {
     setShowAnswer((prev) => !prev);
   };
   return (
-    <div class="scrollbar-thin mb-16 cursor-pointer  " onClick={handleClick}>
-      <div class="flex justify-between">
-        <p class={`${textColor} text-xl font-semibold hover:text-primary`}>
-          {props.title}
-        </p>
-        <button>
-          {!showAnswer && (
-            <svg
-              class={`w-5 h-5 ${textColor}`}
-              fill="none"
-              stroke-linecap="round"
-              stroke-linejoin="round"
-              stroke-width="2"
-              viewBox="0 0 24 24"
-              stroke="currentColor"
-            >
-              <path d="M19 9l-7 7-7-7"></path>
-            </svg>
-          )}
-          {showAnswer && (
-            <svg
-              class={`w-5 h-5 ${textColor} transform rotate-180`}
-              fill="none"
-              stroke-linecap="round"
-              stroke-linejoin="round"
-              stroke-width="2"
-              viewBox="0 0 24 24"
-              stroke="currentColor"
-            >
-              <path d="M19 9l-7 7-7-7"></path>
-            </svg>
-          )}
-        </button>
+    <>
+      <div class="scrollbar-thin mb-16 cursor-pointer  " onClick={handleClick}>
+        <div class="flex justify-between">
+          <p class={`${textColor} text-xl font-semibold hover:text-primary`}>
+            {props.title}
+          </p>
+          <button>
+            {!showAnswer && (
+              <svg
+                class={`w-5 h-5 ${textColor}`}
```

Comment:
```suggestion
                class={`w-5 h-5 ${textColor}`}
```
Should use ternary operator on the class name instead of the svg element.